code/modules/research/nanites/nanite_chamber.dm

```diff
@@ -25,8 +27,16 @@
 
 /obj/machinery/nanite_chamber/RefreshParts()
 	scan_level = 0
+	nanite_coeff = 0
+	speed_coeff = 1
 	for(var/obj/item/stock_parts/scanning_module/P in component_parts)
 		scan_level += P.rating
+	for(var/obj/item/stock_parts/manipulator/manipulator in component_parts)
+		nanite_coeff += manipulator.rating
```

Comment:
```suggestion
		nanite_coeff += 0.75 + (manipulator.rating * 0.25)
```

Considering this is a direct multiplier to the end output, I think this should scale far less per upgrade, this changes the maximum output to 175% of current instead of 400%.